File: backend/src/main/java/com/pjatk/QuizzApp/Question/QuestionDTO.java
package com.pjatk.QuizzApp.Question;

import com.pjatk.QuizzApp.Answer.AnswerDTO;
import com.pjatk.QuizzApp.Quiz.Quiz;
import lombok.Data;

import java.util.List;
import java.util.Set;

@Data
public class QuestionDTO
{
    private String question;
    private List<AnswerDTO> answers;
    private Integer quizId;
}

File: frontend/src/types/question.dto.ts
import type { QuestionType,  } from '@/generated/graphql.ts'
import type { AnswerDto } from '@/types/answer.dto.ts'



export type QuestionDto =
  {
    id: number;
  answers: AnswerDto[];
  questionText: string
  questionType: QuestionType;
  quizId : number;
};
